timings/helpers.go

```diff
@@ -34,6 +34,11 @@ func NewForTask(task *state.Task) *Timings {
 	return New(tags)
 }
 
+// SetTagFromChange sets "change-id" tag from the given change.
+func (t *Timings) SetTagFromChange(change *state.Change) {
```

Comment:
this feels too specific,  I would just do the one liner directly and have the helper, also because is unclear if all places would want to use just "change-id" as tag name